### PR TITLE
(DOCS-3530) Add docker environment variable to Agent docs

### PR DESCRIPTION
**Pull request by hestonhoffman**
See [DOCS-3530](https://datadoghq.atlassian.net/browse/DOCS-3530).

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/agent/docker/_index.md [hunk 1]
```diff
@@ -145,6 +145,7 @@ The Agent's [main configuration file][13] is `datadog.yaml`. For the Docker Agen
 | `DD_URL` (6.36+/7.36+)            | Alias for `DD_DD_URL`. Ignored if `DD_DD_URL` is already set.                                                                                                                                                                                                                                                                                    |
 | `DD_CHECK_RUNNERS`   | The Agent runs all checks concurrently by default (default value = `4` runners). To run the checks sequentially, set the value to `1`. If you need to run a high number of checks (or slow checks), the `collector-queue` component may fall behind and fail the health check. You can increase the number of runners to run checks in parallel. |
 | `DD_APM_ENABLED`             | Enables trace collection. Defaults to `true`. Fore more information about additional trace collection environment variables, see [Tracing Docker Applications][14].   |
+| `DD_LOGS_CONFIG_EXPECTED_TAGS_DURATION` | In some environments, the initial logs from hosts might not include the correct tags. If you're experiencing missing tags in your logs, include this environment variable and set it to `"10m"`.|
```
Comment (maycmlee) [inline content/en/agent/docker/_index.md:148]:
```suggestion
| `DD_LOGS_CONFIG_EXPECTED_TAGS_DURATION` | In some environments, the initial logs from hosts might not include the correct tags. If you're missing tags in your logs, include this environment variable and set it to `"10m"`.|
```

**Review by maycmlee — APPROVED** [review]
Small nit!

File: content/en/agent/docker/_index.md (hunk 1 above)
Comment (djmitche) [reply content/en/agent/docker/_index.md:148]: Maybe "If you're missing tags on new hosts"?  Just an idea :)

**Review by djmitche — APPROVED** [review]
Looks good - take or leave the comment :)

**Comment by hestonhoffman** [comment]
Great, thanks!

**Pull request merged**